internal/config/variables/variables.go

```diff
@@ -400,8 +400,8 @@ func EvaluateVariables(
 
 	// check that all variables have a set value, including default of null
 	for name := range vs {
-		_, found := iv[name]
-		if !found {
+		value := iv[name]
+		if value == nil {
```

Comment:
Is it possible for a variable be found, but nil? It looks like we're not checking for that case anymore with this change.

Reply:
If that's the case, a common pattern I use is:

```go
v, ok := m[k]
if !ok || v == nil {
```

Reply:
If the value of a map is nullable, I usually do the above anyways out of caution.

Reply:
@izaaklauer yes, the case we are checking for here is if a found variable is nil. At this point, we have checked for "non found" variables already. But, to your and Mitchell's points, caution is good! I'll incorporate Mitchell's suggestion.